Installer/Inno_Setup_Compiler_Script.iss

```diff
@@ -15,12 +12,12 @@ AppPublisher={#MyAppPublisher}
 AppPublisherURL={#MyAppURL}
 AppSupportURL={#MyAppURL}
 AppUpdatesURL={#MyAppURL}
-DefaultDirName={localappdata}\Finkit\ManicTime\Plugins\MFitzpatrick.TagSource.Azure7PacePlugin
+DefaultDirName={localappdata}\Finkit\ManicTime\Plugins\Packages\MFitzpatrick.TagSource.Azure7PacePlugin
 DefaultGroupName={#MyAppName}
 AllowNoIcons=yes
-OutputDir=C:\Users\{USERNAME}\Downloads\Azure-Manic-7Pace-master
+OutputDir=C:\Users\Toby.Goulden\Downloads\Azure-Manic-7Pace-master
```

Comment:
You can't just assume everyone has the same name 😛

Reply:
Removed username from the file path.